Assets/Scripts/Gamestate/PlayerManager.cs

```diff
@@ -230,7 +230,9 @@ public void SetLayer(int playerIndex)
         // Set layers for the camera to ignore (the other players' gun layers, and this layer)
         // Bitwise negation of this player's model layer and all gun layers that do not belong to this player
         // Gun layers are 4 above their respective player layers.
-        var playerAndGunMask = ((1 << playerLayer) | allGunsMask) & ~(1 << (playerLayer + 4));
+        var playerMask = 1 << playerLayer;
+        var gunMask = (1 << (playerLayer + 4)) ^ allGunsMask;
```

Comment:
Oh, I meant `1 << (playerLayer + 4)` specifically, but this works too. Perhaps we could put of the OR-ing to the `var negatedMask =` line?

Reply:
Will do that too 👍